R/glue_apa_results.R

```diff
@@ -71,7 +72,7 @@ glue_apa_results <- function(x = NULL, ...) {
     )
 
     if(!is.null(x) && is.data.frame(x)) {
-        if(!is.null(x$term)) x$term <- prettify_terms(x$term)
+        # if(!is.null(x$term)) x$term <- prettify_terms(x$term) BUGFIX: applied prettify_terms() twice
```

Comment:
I understand the function is applied twice with your workflow, but if we go through `broom`, which I would still prefer, it would still need to be applied here, no?

Reply:
I also wondered what might be the best way to create a consistent workflow here. I concluded that the internal function `beautify()` does typesetting of all columns of the `apa_results_table`, and it should therefore also typeset the `$term` column. A similar line of thought applies to `glue_apa_results()`: In the examples section, you manually typeset the columns that are not processed with the standard `printnum_transformer`, and I think this should also happen with the `$term` column, which is why I exported `prettify_terms()`. However, I also believe that the function name really is not perfect, and I would opt for something like `print_terms()` (because it does typesetting like `printnum()`, `print_df()` or `printp()`). As an alternative, maybe it's a good idea to include default typesetting facilities in `glue_apa_results()` (e.g. guessing which one of the functions `printp()`, `printnum()`, `print_df()` or `print_terms()` should be applied to each column). If we provided these facilities, we could refrain from exporting `prettify_terms()`

Reply:
Would it be possible to revert this change, so we can have this discussion in a separate issue/PR?

Reply:
Sure, good point.